exporter/opentelemetry-exporter-jaeger-proto-grpc/src/opentelemetry/exporter/jaeger/proto/grpc/__init__.py

```diff
@@ -161,25 +166,16 @@ def export(self, spans) -> SpanExportResult:
         jaeger_spans = translator._translate(pb_translator)
         batch = model_pb2.Batch(spans=jaeger_spans)
         request = PostSpansRequest(batch=batch)
-        self._collector_grpc_client.PostSpans(request)
-
-        return SpanExportResult.SUCCESS
+        try:
+            self._collector_grpc_client.PostSpans(
+                request, timeout=self._timeout
+            )
+            return SpanExportResult.SUCCESS
+        except RpcError as error:
```

Comment:
Were we incorrectly not catching this exception before?

Reply:
Yes, We werent' doing anything earlier so this was being handled in processor.